src/ydata_synthetic/synthesizers/regular/model.py

```diff
@@ -14,6 +14,7 @@
 from ..regular.cwgangp.model import CWGANGP
 from ..regular.cramergan.model import CRAMERGAN
 from ..regular.dragan.model import DRAGAN
+from ..regular.ctgan.model import CTGAN
```

Comment:
Can we change as well these imports for absolute ones please?

Reply:
Done.